src/backend/drm/device/mod.rs

```diff
@@ -336,7 +361,16 @@ where
                 for event in events {
                     if let Event::PageFlip(event) = event {
                         trace!(self.logger, "Got a page-flip event for crtc ({:?})", event.crtc);
-                        callback(DrmEvent::VBlank(event.crtc), &mut ());
+                        let metadata = EventMetadata {
+                            time: if self.has_monotonic_timestamps {
+                                Time::Monotonic(unsafe { std::mem::zeroed::<Instant>() } + event.duration)
```

Comment:
Comment here would be helpful, adding something to zeroed Instant is non-obvious operation.

Reply:
That is fair, added.